public/app/features/serviceaccounts/ServiceAccountPage.tsx

```diff
@@ -70,7 +70,9 @@ export const ServiceAccountPageUnconnected = ({
 
   const serviceAccountId = parseInt(match.params.id, 10);
   const tokenActionsDisabled =
-    !contextSrv.hasPermission(AccessControlAction.ServiceAccountsWrite) || serviceAccount.isDisabled;
+    serviceAccount.isDisabled ||
```

Comment:
Question: should we lock disabled service accounts? Or should we allow, for instance, deleting a token from a disabled service account 🤔

Reply:
We don't want to disable tokens within this scope. For now, we should allow the user to disable the service accounts (or the plugin itself) instead. As a second step, we might allow the rotation of a token; but that's out of scope.

Reply:
My question was kind of out of the scope of this PR - what I meant is that we don't allow carrying out token actions if `serviceAccount.isDisabled` (this was already the case before your changes). And I was curious about the thinking behind that. Nothing that needs answering now, it was just me being curious.

Reply:
Oh! Thanks for the clarification and sorry for the misinterpretation. I believe it makes sense to delete all active tokens upon disabling the service account, and encourage the user to create new tokens when needed.

@IevaVasiljeva what do you think is the best course of action?